### PR TITLE
feat: add maps to gas/maintenance/paint shops pages

### PR DESCRIPTION
**Pull request by aeassaf**
- Added mapbox's map to Gas, Maintenance and Paint shops pages

### CODE REVIEW

**Comment by aeassaf** [comment]
Capture

Gas - Maintenance - Paint Shops pages now looks like this

File: src/components/Map/index.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const Map = () => (
+  <MapGL
+    width={700}
+    height={450}
+    latitude={37.768}
+    longitude={-122.331}
+    zoom={9.017}
+    mapStyle="mapbox://styles/mapbox/dark-v9"
+    mapboxApiAccessToken="pk.eyJ1IjoiYW50b2luZS1hc3NhZjE1IiwiYSI6ImNqeW8wcjFseTB0bDkzbnBlc2V1aGRjdnUifQ.yfNdGKv27Nxa4lDiZ5pwtQ"
```
Comment (george-haddad) [inline src/components/Map/index.jsx]: Add this as an env variable and have the real key in your netlify deploy

**Review by george-haddad — CHANGES_REQUESTED** [review]
Need to add env var on mapbox key

File: src/components/Map/index.jsx (hunk 1 above)
Comment (aeassaf) [reply src/components/Map/index.jsx]: @george-haddad I think I fixed the issue!

**Review by george-haddad — APPROVED** [review]
👍

**Pull request merged**